src/util/futurepool.rs

```diff
@@ -22,11 +22,28 @@ use std::thread;
 use std::time::Duration;
 use futures::Future;
 use futures_cpupool::{self as cpupool, CpuFuture, CpuPool};
+use prometheus::{IntCounter, IntCounterVec, IntGauge, IntGaugeVec};
 
 use util;
 use util::time::Instant;
 use util::collections::HashMap;
 
+lazy_static! {
+    pub static ref FUTUREPOOL_PENDING_TASK_VEC: IntGaugeVec =
+        register_int_gauge_vec!(
+            "tikv_futurepool_pending_task_total",
+            "Total number of future_pool pending tasks.",
+            &["prefix"]
+        ).unwrap();
+
+    pub static ref FUTUREPOOL_HANDLED_TASK_VEC: IntCounterVec =
+        register_int_counter_vec!(
+            "tikv_futurepool_handled_task_total",
+            "Total number of future_pool handled tasks.",
+            &["prefix"]
```

Comment:
I think `pool` is better than `prefix`.

Reply:
I've changed to `"name"` to keep sync with Worker.